pages/index.tsx

```diff
@@ -132,12 +140,17 @@ export default function Home({
                   <div className="text-red-500">Lahjansaaja on pakollinen</div>
                 )}
               </div>
-              <Button
-                type="submit"
-                className={`mt-8 ${isFetching || isError ? 'cursor-not-allowed bg-red-500' : null}`}
-                disabled={isFetching || isError}
-              >
+              <Button type="submit" className="mt-8" disabled={isFetching}>
```

Comment:
`disabled={isFetching}` pitäisi olla seuraavasti

```suggestion
              <Button type="submit" className="mt-8" disabled={isFetching || createGiftQuery.isPending}>
```

Ehdin jo ruveta lisäämään kommentteja niin en pushannut sitä vielä. Nämä kommentit menevät "piiloon", muuten koska olisivat "Outdated"